rtbkit/core/router/filters/testing/creative_filters_test.cc

```diff
@@ -215,3 +215,92 @@ BOOST_AUTO_TEST_CASE( testSegmentsFilter )
     check(filter, br3, creatives, 0, { {1}, {2}         });
 
 }
+
+
+/******************************************************************************/
+/* PMP FILTER                                                              */
+/******************************************************************************/
+
+BOOST_AUTO_TEST_CASE( testiPMPFilter )
+{
+    CreativePMPFilter filter;
+    CreativeMatrix creatives;
+    auto addDeal = [] (AgentConfig& cfg, const std::string& dealId) {
+        for ( auto& c : cfg.creatives)
+            c.dealId = dealId;
+    };
+  
+    auto addImpr = [] (BidRequest& br,OpenRTB::AdPosition::Vals pos, OpenRTB::PMP& pmp) {
```

Comment:
I'd make this lambda name a little more distinct because it can easily be confused with the addImp function. Something like addPmpImp would do nicely.